packages/web-app-external/src/index.js

```diff
@@ -15,13 +15,13 @@ const appInfo = {
 const routes = [
   {
     name: 'apps',
-    path: '/:file_id/:app?',
+    path: '/:contextRouteName/:file_id/:app?',
```

Comment:
Added the contextRouteName so that we can check the auth requirements of the context route (and redirect to IDP login page if coming from a private context).